The Economic Times.js

```diff
@@ -0,0 +1,192 @@
+{
+	"translatorID": "1a9a7ecf-01e9-4d5d-aa19-a7aa4010da83",
+	"label": "The Economic Times",
+	"creator": "Sonali Gupta",
+	"target": "http://economictimes.indiatimes.com",
```

Comment:
I thought since this section is generated automatically, so no changes were to be made. Thanks for letting me know.